biome.jsonc

```diff
@@ -20,11 +20,12 @@
     "rules": {
       "recommended": true,
       "style": {
-        "noUselessElse": "off", // TODO: turn on, too many cases
-        "noNonNullAssertion": "off" // TODO: turn on, too many cases
+        "noUselessElse": "error",
+        "noNonNullAssertion": "error"
```

Comment:
Let's not enable `noNonNullAssertion` as too many cases that we can handle better (it's ok to lint ignore sometimes)